### DISCUSSION TITLE
Lima on Termux

### DISCUSSION THREAD

**Discussion by wviana** [lima-vm/lima#768] [Q&A]
Hello,

Today I just wonder if I couldn't install Lima on Termux,
Just checked for qemu packages on it, Installed `qemu-system-aarch64`.
Ok, now Just have to install colima. Found this command in README.

```shell
brew install jq
VERSION=$(curl -fsSL https://api.github.com/repos/lima-vm/lima/releases/latest | jq -r .tag_name)
curl -fsSL "https://github.com/lima-vm/lima/releases/download/${VERSION}/lima-${VERSION:1}-$(uname -s)-$(uname -m).tar.gz" | tar Cxzvm /usr/local
```

Did just some tweak in it. Installed `jq` with apt ( could be pkg ) and changed to extract in "termux /usr/local". It looked like this:
```shell
apt install jq  # Could be pkg i jq, but pkg always does update before install,
VERSION=$(curl -fsSL https://api.github.com/repos/lima-vm/lima/releases/latest | jq -r .tag_name)
curl -fsSL "https://github.com/lima-vm/lima/releases/download/${VERSION}/lima-${VERSION:1}-$(uname -s)-$(uname -m).tar.gz" | tar Cxzvm "/data/data/com.termux/files/usr/local"
```
So it installed lima on termux.

Did it worked? 
*NO* :cry: 

Even getting autocomplete for limactl command and been sure it is on path. I'm getting `No command limactl found, did you mean...`. Already `rehash`ed, close termux and open again even navigate to bin directory and try to run as `./limactl`. From bin directory I get a `zsh: no such file or directory: ./limactl`

Does anyone could help me on this?
My intention is to be able to run k3s on termux using lima.
If it works we could create a termux package for it.

**Comment by wviana** [comment]
Here is which and type outputs.
```shell
$which limactl
/data/data/com.termux/files/usr/bin/limactl
$type limactl
limactl is /data/data/com.termux/files/usr/bin/limactl
```

**Comment by wviana** [comment]
Tried to compile in termux instead of trying to use binary.
Could install it. But am getting [this error](https://github.com/termux/termux-app/discussions/2738#discussioncomment-2635568) when trying to run `limactl start` does anyone could help me on this?

**Comment by wviana** [comment]
If anyone reach here looking for a way to run lima inside termux this discussion os how much progress was done so far. 

https://github.com/termux/termux-app/discussions/2738#discussioncomment-2635568

**Comment by luisdavim** [comment]
looks like there has been some progress on this: https://github.com/termux/termux-packages/issues/19658#issuecomment-2034152039
